src/main/groovy/wooga/gradle/unity/UnityTask.groovy

```diff
@@ -82,7 +94,8 @@ abstract class UnityTask extends DefaultTask
      */
     protected void preExecute() {
         if (projectDirectory.present && !isSet(UnityCommandLineOption.projectPath)) {
-            projectPath = projectDirectory.get().asFile.path
+            def _path = projectDirectory.get().asFile.path
+            projectPath = _path
```

Comment:
Why the two lines? I guess you needed to debug a value?

Reply:
I was transforming the value but ended up taking that back. Will revert.